Cake.Recipe/Content/travisci.cake

```diff
@@ -0,0 +1,41 @@
+BuildParameters.Tasks.PrintTravisCIEnvironmentVariablesTask = Task("Print-TravisCI-Environment-Variables")
+    .WithCriteria(() => TravisCI.IsRunningOnTravisCI)
+    .Does(() =>
+    {
+        using(var travisCI = TravisCI.Fold("Environment Variables"))
+        {
+            Information("CI: {0}", EnvironmentVariable("CI"));
+            Information("TRAVIS_BRANCH: {0}", EnvironmentVariable("TRAVIS_BRANCH"));
+            Information("TRAVIS_BUILD_DIR: {0}", EnvironmentVariable("TRAVIS_BUILD_DIR"));
+            Information("TRAVIS_BUILD_ID: {0}", EnvironmentVariable("TRAVIS_BUILD_ID"));
+            Information("TRAVIS_BUILD_NUMBER: {0}", EnvironmentVariable("TRAVIS_BUILD_NUMBER"));
+            Information("TRAVIS_COMMIT: {0}", EnvironmentVariable("TRAVIS_COMMIT"));
+            Information("TRAVIS_COMMIT_RANGE: {0}", EnvironmentVariable("TRAVIS_COMMIT_RANGE"));
+            Information("TRAVIS_JOB_ID: {0}", EnvironmentVariable("TRAVIS_JOB_ID"));
+            Information("TRAVIS_JOB_NUMBER: {0}", EnvironmentVariable("TRAVIS_JOB_NUMBER"));
+            Information("TRAVIS_OS_NAME: {0}", EnvironmentVariable("TRAVIS_OS_NAME"));
+            Information("TRAVIS_PULL_REQUEST: {0}", EnvironmentVariable("TRAVIS_PULL_REQUEST"));
+            Information("TRAVIS_REPO_SLUG: {0}", EnvironmentVariable("TRAVIS_REPO_SLUG"));
+        }
+    });
+
+BuildParameters.Tasks.UploadTravisCIArtifactsTask = Task("Upload-TravisCI-Artifacts")
```

Comment:
This task should maybe perhaps have another criteria, let us say that you've got both travis and appveyor setup to run on every commit and tag, if the project is written in c# it isn't usually necessary to upload packages from both of them.

So what I'm suggesting is to have a criteria that would decide if a package should be uploaded from travis or from appveyor (or perhaps even both).

Hope that makes sense.

Reply:
yup, sounds good to me.

I noticed first after I had commented that it was a WIP 😕

Reply:
No worries.  That is the reason for WIP, so that I could get feedback and make sure I stay on the right path.  😄 
Keep the comments and thoughts coming!